lib/addons/vpc-cni/index.ts

```diff
@@ -275,6 +275,13 @@ export interface VpcCniAddOnProps {
    */
   serviceAccountPolicies?: iam.IManagedPolicy[];
 
+  /**
+   * Enable kubernetes network policy in the VPC CNI introduced in vpc-cni 1.14
+   * More informaton on official AWS documentation: https://docs.aws.amazon.com/eks/latest/userguide/cni-network-policy.html
+   * 
+   */
+  enableNetworkPolicy?: string;
```

Comment:
Minor comment: why is it a string? Let's make it a boolean, then it is clear and we don't have to handle cases like "FALSE" vs "false".